activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -7,7 +7,8 @@ module Reporting
       attr_accessor :gem_name
 
       # Outputs a deprecation warning to the output configured by
-      # <tt>ActiveSupport::Deprecation.behavior</tt>.
+      # <tt>ActiveSupport::Deprecation.behavior</tt>. By default
+      # +callstack+ is equal to caller(2).
```

Comment:
I think it's fine as is: `callstack` is optional because it's quite rare that you need to specify it. And I think anyone capable of working out they need to override the callstack is going to know that the way to get a callstack in ruby is from `caller` (or, given their unusual requirement, maybe from an exception?).